### ISSUE TITLE
Using --noserver picks up no issues and returns Passed even when there are issues

### ISSUE THREAD

**Issue by colossusb** [nvuillam/npm-groovy-lint#290]
**Summary**

Running with '--noserver' does not pick up issues yet everything appears Passed
This has been run on serveral different machines and always replicates as below.
Wondering if perhaps some CodeNarc update has broken this?

Might be related to https://github.com/nvuillam/npm-groovy-lint/issues/284 ?

**To Replicate**

1. Make some obvious error in your code
2. Run 'npm-groovy-lint <filename>' to confirm the error is picked up
3. Run 'npm-groovy-lint <filename> --noserver' and witness the error is not picked up

**Versions**
```
npm-groovy-lint --version
GroovyLint: Successfully processed CodeNarc: 
CodeNarc version 3.1.0

npm-groovy-lint version 11.1.1

Embeds:
CodeNarc version 3.1.0
- Groovy version 3.0.9 (superlite)
```

**OS's**
Ventura
Linux

**Example**
```
me@mymachine my_folder % npm-groovy-lint ./vars/addSshAuthorizedKeys.groovy           
/Users/me/PycharmProjects/my_folder/vars/addSshAuthorizedKeys.groovy
  10    error    Unexpected input: '[\n        whoops_I_put_a_Bunch_of_Crap_in_My_Code\n        'publicKey'' @ line 10, column 9.  NglParseError           


npm-groovy-lint results in 1 linted files:
┌─────────┬───────────┬─────────────┐
│ (index) │ Severity  │ Total found │
├─────────┼───────────┼─────────────┤
│    0    │  'Error'  │      1      │
│    1    │ 'Warning' │      0      │
│    2    │  'Info'   │      0      │
└─────────┴───────────┴─────────────┘
Failure: 1 error(s) have been found 
 0 warning(s) have been found 
 0 info(s) have been found
me@mymachine my_folder % npm-groovy-lint ./vars/addSshAuthorizedKeys.groovy --noserver

npm-groovy-lint results in 0 linted files:
┌─────────┬───────────┬─────────────┐
│ (index) │ Severity  │ Total found │
├─────────┼───────────┼─────────────┤
│    0    │  'Error'  │      0      │
│    1    │ 'Warning' │      0      │
│    2    │  'Info'   │      0      │
└─────────┴───────────┴─────────────┘
```

**Comment by yermulnik** [comment]
Same here 😢 
```shell
> npm ls -g
/home/giermulnik/.npm-global/lib
└── npm-groovy-lint@11.1.1

> npm-groovy-lint --noserver ./test.jenkinsfile

npm-groovy-lint results in 0 linted files:
┌─────────┬───────────┬─────────────┐
│ (index) │ Severity  │ Total found │
├─────────┼───────────┼─────────────┤
│    0    │  'Error'  │      0      │
│    1    │ 'Warning' │      0      │
│    2    │  'Info'   │      0      │
└─────────┴───────────┴─────────────┘
```

**Comment by tbroyer** [comment]
Running into the same problem as well. This is indeed #284 and the analysis here is correct: this code needs to be applied in `callCodeNarcJava` as well as `callCodeNarcServer`:

https://github.com/nvuillam/npm-groovy-lint/blob/6d7806b23256581740b5447a7b9cdc6a9565674f/lib/codenarc-caller.js#L62-L72

## Workaround

Pass dummy arguments before and after your file list:
```
$ npm-groovy-lint --noserver dummy ./vars/addSshAuthorizedKeys.groovy dummy
```

**Comment by tbroyer** [comment]
And/or maybe npm-groovy-lint should not include quotes in the `-includes` parameter when it builds it?

https://github.com/nvuillam/npm-groovy-lint/blob/6d7806b23256581740b5447a7b9cdc6a9565674f/lib/codenarc-factory.js#L117

**Comment by nvuillam** [comment]
Would anyone here submit a PR with the fix ? :)

**Comment by nvuillam** [comment]
This problem might be solved in v13.0.0, do you still have it ?

**Comment by tbroyer** [comment]
I confirm this now works as expected with 13.0.0.

**Comment by nvuillam** [comment]
Thanks a lot for your feedback :)
